lib/cuda/cudaCopyFromRingbuffer.cpp

```diff
@@ -126,9 +126,15 @@ cudaEvent_t cudaCopyFromRingbuffer::execute(cudaPipelineState& pipestate,
     auto tmp = std::make_shared<chordMetadata>();
     tmp->deepCopy(meta);
     meta = tmp;
+    assert(meta->get_fpga_seq_num() == 0);
     assert(meta->get_sample0_offset() == 0);
     assert(input_cursor % meta->sample_bytes() == 0);
-    meta->set_sample0_offset(meta->get_sample0_offset() + input_cursor / meta->sample_bytes());
+    meta->set_fpga_seq_num(meta->get_fpga_seq_num()
+                           + meta->get_time_downsampling_fpga().at(0)
```

Comment:
this is not done quite consistently, sometimes it does `fpga_seq_num = time_downsamplng_fpga * (input_cursor / sample_bytes)` like here, and sometimes it misses the `time_downsampling_fpga` like in https://github.com/kotekan/kotekan/pull/1384/changes#diff-31ced5ba1ee248fe32f1668ff1ffa2fc6abaf6752420acd4bb05754974a71e4dR233